services/sharedchannel/service.go

```diff
@@ -177,3 +178,16 @@ func (scs *Service) pause() {
 
 	scs.server.GetLogger().Debug("Shared Channel Service inactive")
 }
+
+// Makes the remote channel to be read only, only for guests.
+func (scs *Service) makeChannelReadOnly(channel *model.Channel) *model.AppError {
+	readonlyChannelModerations := []*model.ChannelModerationPatch{
+		{
+			Name:  &model.ChannelModeratedPermissions[0],
```

Comment:
Why not just have `PERMISSION_CREATE_POST.Id`? Easy to remember and read.

Reply:
For sure, will amend

Reply:
amended in https://github.com/mattermost/mattermost-server/pull/16803/commits/749ec630122e22d8f49487d7c88457cbc7ba70f4